backend/src/test/java/com/redhat/cloud/notifications/db/BehaviorGroupResourcesTest.java

```diff
@@ -211,7 +210,7 @@ public void testAddMultipleEmailSubscriptionBehaviorGroupActions() {
         BehaviorGroup behaviorGroup = createBehaviorGroup("displayName", bundle.getId());
         Endpoint endpoint1 = createEndpoint(EMAIL_SUBSCRIPTION);
         Endpoint endpoint2 = createEndpoint(EMAIL_SUBSCRIPTION);
-        updateAndCheckBehaviorGroupActions(ACCOUNT_ID, bundle.getId(), behaviorGroup.getId(), BAD_REQUEST, endpoint1.getId(), endpoint2.getId());
+        updateAndCheckBehaviorGroupActions(ACCOUNT_ID, bundle.getId(), behaviorGroup.getId(), OK, endpoint1.getId(), endpoint2.getId());
```

Comment:
This test doesn't really make sense now. Should it be deleted?

Reply:
Not a strong opinion about it.
 
Before we didn't allow multiple Email endpoints, now we do, so I don't think it makes any harm in keeping it to ensure we still allow multiple Email endpoints.